pkg/controller/directvolumemigration/migrate.go

```diff
@@ -34,7 +38,7 @@ func (r *ReconcileDirectVolumeMigration) migrate(direct *migapi.DirectVolumeMigr
 	}
 	err = task.Run()
 	if err != nil {
-		if errors.IsConflict(err) {
+		if k8serrors.IsConflict(err) {
```

Comment:
Will have to use unwrap here. Please wait for #902 to get in

Reply:
Done